components/automate-minio/habitat/hooks/run

```diff
@@ -2,9 +2,6 @@
 
 exec 2>&1
 
-chmod 0600 "{{pkg.svc_config_path}}/secrets"
```

Comment:
This implies that we'll no longer allow people to set credentials in the config. It's probably best to let the credential resolvers look for AWS creds in the standard location but I'm not sure if this is going to break people or not.

Reply:
those values were never set. The file was copied and pasted from the backup-gateway config:
```
[94][default:/src:0]# cat /hab/svc/automate-minio/config/secrets 

[95][default:/src:0]# 

```

Reply:
```
{{~#if cfg.gateway.backup.s3.credentials.access_key}}
export AWS_ACCESS_KEY_ID="{{cfg.gateway.backup.s3.credentials.access_key}}"
{{~/if}}
{{~#if cfg.gateway.backup.s3.credentials.secret_key}}
export AWS_SECRET_ACCESS_KEY="{{cfg.gateway.backup.s3.credentials.secret_key}}"
{{~/if}}
{{~#if cfg.gateway.backup.s3.credentials.session_token}}
export AWS_SESSION_TOKEN="{{cfg.gateway.backup.s3.credentials.session_token}}"
{{~/if}}
```

Thats the contents of the deleted file. It's impossible to set those values as this is not the backup gateway

Reply:
:facepalm: I thought I was looking at the bgw